sipa/blueprints/usersuite.py

```diff
@@ -171,7 +171,7 @@ def generate_epc_qr_code(details: PaymentDetails, months):
         bic=details.bic.replace(' ', ''),
         recipient=details.recipient,
         iban=details.iban.replace(' ', ''),
-        amount=months * MEMBERSHIP_CONTRIBUTION / 100,
+        amount=months * current_app.config['MEMBERSHIP_CONTRIBUTION'] / 100,
```

Comment:
Just to have mentioned it, in python we can use double or single quotes – however, with the advent of new tools such as the [ `black`](https://black.readthedocs.io/en/stable/) autoformatter, it has been decided to prefer double quotes (`"`). There's no particular reason other than to have a preference, so you don't have to always choose between the two.

But this is not important until someone adds `black` (or rather [`darker`](https://pypi.org/project/darker/)) as a pre-commit hook.